example/pubspec.yaml

```diff
@@ -39,6 +39,7 @@ dependencies:
 
   wagmi_flutter_web:
     path: ../
+  # web3dart: ^2.7.3
```

Comment:
Why this declaration of web3dart (just for info webthree is better :))

Reply:
Sorry my bad! We were using it for testing purposes.